### PR TITLE
Update SpeedrunCommand.scala

### PR DESCRIPTION
**Pull request by Fezehim**

### CODE REVIEW

File: src/me/krickl/memebotj/InternalCommands/SpeedrunCommand.scala [hunk 1]
```diff
@@ -109,6 +109,8 @@ class SpeedrunCommand(channel: String, command: String, dbprefix: String) extend
     this.runnerList.add("Ur Mom")
     
     this.runnerList.add("{runner}")
+    
+     this.runnerList.add("Sadiecat122")
```
Comment (unlink2) [inline src/me/krickl/memebotj/InternalCommands/SpeedrunCommand.scala]: Bad formatting: Unnecessary whitespace in line 113.

Comment (Fezehim) [inline src/me/krickl/memebotj/InternalCommands/SpeedrunCommand.scala]: shit, didn't notice

**Comment by unlink2** [comment]
Will merge anyway
